### DISCUSSION TITLE
Link branch to an issue.

### DISCUSSION THREAD

**Discussion by Morpheus636** [community/community#4920] [Projects and Issues] [answered]
There should be a way to link a branch to an issue, just like you can link a pull request, so you can easily check on the development of a feature, bugfix, etc.

**Comment by daanleenders** [comment]
We would like to use something like this too. Although we have a pretty decent work around. We usually open a draft PR that points you to the branch. It is a couple of clicks extra but works decently enough.

**Reply by ghost** [reply]
We also would love this, especially if it included something like expanded automation for when a branch is created

**Comment by Morpheus636** [comment, marked as answer]
Looks like it's been added! Thanks GitHub.
